code/modules/mob/living/simple_animal/hostile/retaliate/undead.dm

```diff
@@ -94,6 +94,43 @@
 	del_on_death = TRUE
 	footstep_type = FOOTSTEP_MOB_SHOE
 
+/mob/living/simple_animal/hostile/retaliate/skeleton/warden
+	name = "skeleton warden"
+	desc = "The remains of a warden. A keycard rests on its belt."
```

Comment:
```suggestion
	desc = "The remains of a warden."
```
That description won't necessarily be true after stealing the keycard. I would either change it up or just remove that segment.